### PR TITLE
Added pip install iota-sdk to python readme as first option

### LINKED ISSUE #1396: Add `pip install iota-sdk` to python readme
## Description

Add `pip install iota-sdk` as first installation option to othe python readme

## Motivation

Easiest way to install it when a wheel is available for this target

## Are you planning to do it yourself in a pull request?

No.

### PR DESCRIPTION
**Pull request by shreykul**
# Description of change

Added pip install iota-sdk to python readme for the ease of installation to the user. 

## Links to any relevant issues

fixes #1396

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project

### CODE REVIEW

File: bindings/python/README.md [hunk 1]
```diff
@@ -46,7 +75,7 @@ Python binding to the [iota-sdk library](/README.md).
    .\env\Scripts\activate`
    ```
 
-3. Install the required dependencies and build the wheel by running the following commands:
```
Comment (Alex6323) [inline bindings/python/README.md:49]: missing something?

File: bindings/python/README.md [hunk 2]
```diff
@@ -24,6 +25,34 @@ Python binding to the [iota-sdk library](/README.md).
...
+   ```bash
+   python3 -m venv env
+   source env/bin/activate
+   ```
+
+   If you are using Windows, you should run the following instead:
+
+   ```bash
+   python3 -m venv env
+   .\env\Scripts\activate`
```
Comment (Thoralf-M) [inline bindings/python/README.md]: I know it was below already like this, but can you remove the ` at the end please? (also below)

Comment (shreykul) [reply bindings/python/README.md]: I removed it from there.

**Review by Thoralf-M — APPROVED** [review]
Thanks!

**Review by kwek20 — APPROVED** [review]

**Pull request merged**